frontend/src/pages/landing.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+
+import Nav from '../components//global/Nav';
```

Comment:
I think there's a typo here on the path having two slashes
I'm a bit surprised that this is not causing an error or warning

Reply:
This has been updated